### DISCUSSION TITLE
api.weather.gov/stations/{stationID}/observations/latest  Out of date

### DISCUSSION THREAD

**Discussion by Yeshia** [weather-gov/api#360] [Q&A]
I just want to make sure I am reading these dates correctly. When I access "api.weather.gov/stations/{stationID}/observations/latest" for my station, the dates all appear to be UNC. But if that is correct, then my Latest data is 2 hours old. I'm noticing the same for other stations I query. 

Is this UNC timestamps in the /observatins/latest response? Is this normal for it to be a couple hours off? Or does that just mean "No Change"? 

Thanks

**Comment by scadergit** [comment]
All dates are UTC (I'm not familar with UNC). The source for observsation data is known to be slow at the moment. Please report these as described here: https://weather-gov.github.io/api/reporting-issues. The more reports sent, the higher priority it will get.

**Comment by Yeshia** [comment]
Yes, UTC is what I meant. Thank you.
